modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -3016,6 +3087,27 @@ private ObjectEntry _addObjectEntry(
 		return objectEntry;
 	}
 
+	private void _assertEmailNotificationSent(
+		int inboxSize, String notificationBody) {
+
+		List<NotificationQueueEntry> notificationQueueEntries =
+			_notificationQueueEntryLocalService.getNotificationEntries(
+				NotificationConstants.TYPE_EMAIL,
+				NotificationQueueEntryConstants.STATUS_SENT);
+
+		Assert.assertEquals(
+			notificationQueueEntries.toString(), inboxSize,
+			notificationQueueEntries.size());
+
+		Assert.assertEquals(inboxSize, MailServiceTestUtil.getInboxSize());
+
+		MailMessage lastMessage = MailServiceTestUtil.getLastMailMessage();
```

Comment:
@algobob rename it to `MailMessage mailMessage` or `MailMessage lastMailMessage`